public/templates/actor/pilot.hbs

```diff
@@ -85,6 +85,19 @@
           <span class="lancer-header major">{{localize "lancer.pilot-sheet.cloud-upload.label"}}</span>
           <span>{{localize "lancer.pilot-sheet.cloud-upload.wip"}}</span>
         </div>
+
+        <div class="card clipped" style="grid-area: 3/1/4/2;">
+          <span class="lancer-header major">{{localize "lancer.pilot-sheet.json-import.label"}}</span>
+          <input id="pilot-json-import" type="file" name="pilot-json-up" class="lcp-up" accept=".json">
+                {{!-- New LCP Manifest --}}
+          {{#if pilotData}}
```

Comment:
Could be wrong here, but I don't believe the pilot sheet's `pilotData` is getting properly passed to the Handlebars context so I think this is always going to be null. At least that's what my testing so far suggests -- I have not yet been able to get the Import button to render after uploading a json file.

Reply:
Yeah I'm not sure of the best way to have handled this, but it's moot now on the current version of the branch.